packages/amplify_auth_cognito/android/src/test/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AmplifyAuthCognitoPluginTest.kt

```diff
@@ -315,6 +317,58 @@ class AmplifyAuthCognitoPluginTest {
         verify(mockResult, times(1)).success(ArgumentMatchers.any<LinkedTreeMap<String, Any>>())
     }
 
+    @Test
+    fun fetchSession_returnsSuccess() {
```

Comment:
Can you remove the type cues in your `val` assignments?

Java had:
```java
LongTypeName type = new LongTypeName();
```
Kotlin allows us to simplify this, dropping one of the type names, a semicolon, and the `new` keyword:
```kotlin
val type = LongTypeName()
```
The type cue can be useful to disambiguate when necessary. But in other cases, it makes the syntax about as bad (maybe a little worse) than Java had been. The below expression is pretty comparable to in Java, just with `new` swapped for `val`, a `;` swapped for `:`, and the remaining items permuted in their ordering:
```kotlin
val type: LongTypeName = LongTypeName()
```

Reply:
@jamesonwilliams - I changed all the versions as indicated.  I also removed the typings on the val assignments throughout the whole file, except for one or two instances.